docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  abdm-wrapper:
+    build: .
+    image: abdm-wrapper
+    ports:
+      - '8080:8080'
```

Comment:
Generally I have noticed people exposing a different port rather than the default port 8080 to avoid conflicts.